main.js

```diff
@@ -17,55 +17,54 @@ console.log(test);
 const arr = [2, 3, 5, 8];
 let res = 1;
 for (let i = 0; i < arr.length; i++) {
-	res *= arr[i] 
+  res *= arr[i];
 }
 console.log(res);
 
 const arr2 = [2, 5, 8, 15, 0, 6, 20, 3];
-	for (let i = 0; i < arr2.length; i++) {
-		if (arr2[i] > 5 && arr2[i] < 10) { 
-			console.log(arr2[i]);
-			}
-	}
+for (let i = 0; i < arr2.length; i++) {
+  if (arr2[i] > 5 && arr2[i] < 10) {
+    console.log(arr2[i]);
+  }
+}
 
 const arr3 = [2, 5, 8, 15, 0, 6, 20, 3];
-	for (let i = 0; i < arr3.length; i++) {
-		if (arr3[i] % 2 === 0) {
-			console.log(arr3[i]);
-		}
-	}	
+for (let i = 0; i < arr3.length; i++) {
+  if (arr3[i] % 2 === 0) {
+    console.log(arr3[i]);
+  }
+}
 
-	//lesson-3
+//lesson-3
 
-const palindromeWord = 'kok';
-const newPal = palindromeWord.split('').reverse().join('');
-function palidrome() {
-	return newPal === palindromeWord;
+function palidrome(palindromeWord) {
+  return palindromeWord === palindromeWord.split("").reverse().join("")
+    ? true
+    : false;
```

Comment:
you still can add new variable inside function with `palindromeWord.split("").reverse().join("")` value and return `palindromeWord === palindromeReverse`